packages/rich-text/src/component/index.js

```diff
@@ -843,21 +843,11 @@ class RichText extends Component {
 		shouldReapply = shouldReapply ||
 			placeholder !== prevProps.placeholder;
 
-		const { activeFormats = [] } = this.record;
-
 		if ( shouldReapply ) {
 			this.value = value;
 			this.record = this.formatToValue( value );
 			this.record.start = selectionStart;
 			this.record.end = selectionEnd;
-
-			updateFormats( {
```

Comment:
So was `updateFormats` mutating the record back to a state with the previous formats? Is that it?

Reply:
Yes, that's it. :) I'm not sure why it's here, I probably made a mistake.